Assets/Scripts/System/GM/GM.cs

```diff
@@ -186,7 +186,7 @@ public void Restart()
         string drive = Environment.GetEnvironmentVariable("HOMEDRIVE");
         string path = Environment.GetEnvironmentVariable("HOMEPATH");
 
-        return Path.Combine(Path.Combine(drive, path), "winnitron.pid");
+        return Path.Combine(drive + path, "winnitron.pid");
```

Comment:
Was there a reason you changed this? `Path.Combine` is here because it's platform-agnostic; "Windows" and "Everyone Else" have different path separator characters.